systems/plants/@RigidBodyManipulator/contactConstraintDerivatives.m

```diff
@@ -1,13 +1,25 @@
-function [n, D, dn, dD] = contactConstraintDerivatives(obj, kinsol, idxA, idxB, xA, xB, normal, d, use_mex)
- compute_second_derivative = nargout > 2;
- 
+function [d, n, D, dn, dD] = contactConstraintDerivatives(use_mex, obj, normal, kinsol, idxA, idxB, xA, xB)
```

Comment:
I did this to make the call that computes only the surface tangents less awkward (i.e. d = contactConstraintsmex(normal) instead of d = contactConstraints(obj.mex_model_ptr, normal))

This is because the surface tangents code doesn't need the mex_model_ptr at all.  I agree this does violate conventions though.  Changed it back.